cmd/antrea-agent/options_windows.go

```diff
@@ -53,6 +53,9 @@ func (o *Options) checkUnsupportedFeatures() error {
 	if encryptionMode != config.TrafficEncryptionModeNone {
 		unsupported = append(unsupported, "TrafficEncryptionMode: "+encryptionMode.String())
 	}
+	if o.config.EnableBridgingMode {
+		unsupported = append(unsupported, "EnableBridgingMode")
```

Comment:
In future release, FlexibleIPAM feature may support Windows. However, it is already `bridging mode` on Windows, although some details and functions are slightly different.
Maybe we need to distinguish between `basic bridging mode on Windows` and `advanced bridging mode with FlexibleIPAM` when naming.

Reply:
`bridging mode` means connecting uplink to bridge + Pod flexible IPAM. Again, if you have a better name, please share.

Reply:
@wenyingd @tnqn Do you have any idea on this topic?